src/__tests__/mountSpec.js

```diff
@@ -327,4 +327,340 @@ describe('UIAtoms mount rendering', () => {
       expect(wrapper.find('noscript').length).toEqual(0);
     });
   });
+
+  describe('should render simple UI atoms: ', () => {
+    let mountOptions = getComponentOptions();
+
+    describe('Headers: ', () => {
+      describe('Headline1: ', () => {
+        it('with text', () => {
+          const { Headline1 } = UIAtoms;
+          wrapper = mount(
+            <PageContext.Provider value={mountOptions.context}>
+              <Headline1 text="text" />
+            </PageContext.Provider>,
+            mountOptions
+          );
+
+          expect(wrapper.html()).toMatchSnapshot();
```

Comment:
Why is `wrapper.html()` needed? Isnt `expect(wrapper).toMatchSnapshot();` enough?